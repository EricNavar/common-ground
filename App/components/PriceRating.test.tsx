import {describe, expect, test} from '@jest/globals';

describe('Test test', () => {
  test('ADD ACTUAL TESTS FOR PriceRating.tsx HERE', () => {
    expect(true).toBe(true);
  });
});