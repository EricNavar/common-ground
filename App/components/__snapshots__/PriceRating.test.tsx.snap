// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PriceRating Renders a Review correctly 1`] = `
<View
  style={
    {
      "alignItems": "flex-start",
      "display": "flex",
      "flexDirection": "row",
      "justifyContent": "flex-start",
    }
  }
>
  <Text>
    $
  </Text>
  <Text
    style={
      {
        "color": "#818181",
      }
    }
  >
    $
  </Text>
  <Text
    style={
      {
        "color": "#818181",
      }
    }
  >
    $
  </Text>
  <Text
    style={
      {
        "color": "#818181",
      }
    }
  >
    $
  </Text>
  <Text
    style={
      {
        "color": "#818181",
      }
    }
  >
    $
  </Text>
</View>
`;
